jiant/models.py

```diff
@@ -1009,6 +1040,21 @@ def _span_prediction_forward(self, batch, task, predict):
             out["preds"] = {"span_start": pred_span_start, "span_end": pred_span_end}
         return out
 
+    def _sop_forward(self, batch, task, predict):
```

Comment:
Do we actually need this? Add a comment to clarify what happens here that isn't already done by _pair_sentence_forward. (The model structure is different, but it seems like that difference lives entirely inside self._get_classifier, right?)

Reply:
I merged it into pair sentence forward.